Packs/FortinetFortiwebVM/Integrations/FortinetFortiwebVM/FortinetFortiwebVM.py

```diff
@@ -1254,8 +1254,20 @@ def error_handler(self, res: Response):
             DemistoException: There is a problem with one or more arguments.
             DemistoException: One or more of the specified fields are invalid. Please validate them.
         """
-        output = res.json()
         error_code = res.status_code
+        if res.status_code == HTTPStatus.UNAUTHORIZED:
+            raise DemistoException(
+                "Authorization Error: make sure Username and Password are set correctly."
+            )
+        if res.status_code == HTTPStatus.NOT_FOUND:
+            raise DemistoException(
+                "Connection Error: Make sure server URL is set correctly."
+            )
+        if res.headers.get("Content-Type") != "application/json":
+            raise DemistoException(
+                res,
+                res=res)
```

Comment:
are these the only possible error codes in our case?
If we are not 100% sure, I'd wrap the next line (with the res.json()) with a try-except block to make sure it can't happen (and raise as much information as I can in case of exception)